source/braille.py

```diff
@@ -1756,6 +1762,17 @@ def _set_windowEndPos(self, endPos):
 					break
 		except ValueError:
 			pass
+		# Prevent that paragraph start indicator is skipped
```

Comment:
Can you explain this comment more? what is the case here

Reply:
When the braille wrap option is enabled, sometimes spaces indicating the start of paragraphs aren't displayed in braille. I'm reading a book in Thorium Reader, a system to read epub and other formats in browse mode, and in specific parts of the book paragraph are displayed without spaces.

Reply:
can you please expand this comment to explain this case further